docs/ids.md

```diff
@@ -1,27 +1,18 @@
-Often, "ID" is synonymous with the auto-incrementing `id` column in your database of choice. For
-most applications, there really is no distinction. But in event sourcing, you need to think of IDs
-as globally-unique identity—separate from the database that you happen to be using at any given time.
+By default, Verbs uses 64-bit integer IDs called "Snowflakes."
 
-## Globally-Unique Identity?
+## Globally Unique Ids
 
-If you create a new table and insert the first row into it, the `id` value is likely to be `1`. This
-is not globally unique in your system. There are other things with an `id` of `1`.
+We do this because an event-sourcing system needs non-sequential, globablly-unique IDs to run well. Replaying events is a massively powerful feature, but does not pair well with standard auto-incrementing IDs.
```

Comment:
They don't need to be specifically non-sequential. In general, Snowflakes are sequential.